### PR TITLE
Uml 118 add data model for lpa

### PR DESCRIPTION
**Pull request by cooperaj**
Implement data entities for the Lpa service. 

Provide a factory class to aid with building the entities from Sirius sourced JSON (as per the Swagger docs)

### CODE REVIEW

File: service-front/app/src/Viewer/src/Service/Lpa/LpaFactory.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+use Viewer\Entity\Lpa;
+use Viewer\Entity\CaseActor;
+use Viewer\Entity\Address;
+use \DateTime;
+use Zend\Stdlib\Exception\InvalidArgumentException;
+
+use function filter_var;
+
+final class LpaFactory
```
Comment (nsmithuk) [inline service-front/app/src/Viewer/src/Service/Lpa/LpaFactory.php]: Why make it final?

Comment (cooperaj) [reply service-front/app/src/Viewer/src/Service/Lpa/LpaFactory.php]: I don't foresee any reason we'll ever need to subclass it. I've always considered that that being the case you should mark things as final.

Heres a good read https://ocramius.github.io/blog/when-to-declare-classes-final/

Comment (cooperaj) [reply service-front/app/src/Viewer/src/Service/Lpa/LpaFactory.php]: As per that article I've clarified the choice by creating an interface for the factory.

Comment (nsmithuk) [reply service-front/app/src/Viewer/src/Service/Lpa/LpaFactory.php]: Cool. Approved.

**Pull request merged**